lib/ransack_ui/ransack_overrides/helpers/form_builder.rb

```diff
@@ -198,6 +198,12 @@ def attribute_collection_for_base(base, attributes=nil)
       rescue UntraversableAssociationError => e
         nil
       end
+
+      private
+
+      def autocomplete_source
+        @autocomplete_source ||= @foreign_klass && @foreign_klass._ransack_autocompletes_through
```

Comment:
right, I've added a rescue, just to keep the `_ransack_autocompletes_through` extracted but not dependent